ci/docker-compose.yml

```diff
@@ -20,14 +20,19 @@ services:
       - APP_URL=https://projectunicorn.net
       - APP_ENV=development
       - WORKSPACES_CONNECTION_STRING=mysql+mysqldb://root:rootpass@db:3306/pub_workspaces?ssl=true
+    volumes:
+    - ../src:/usr/src/app/src
+    command: python -m src.apps.discord.discord_client.py
```

Comment:
Do we need to revert both these lines too since the changes are related to the change in `discordworkspacebot.Dockerfile`?

We may be able to use the `command` to simplify readme instructions - instead of having the developer run `docker exec -it pub-discord-workspace-bot python3 -m src.persistence.schema schema.py` to setup the db we can run it here `command: python3 -m src.persistence.schema schema.py`.

Reply:
tested my suggestion locally - but didn't work as I expected.

Reply:
Removed the command call, kept the volumes to avoid having to rebuild the image every time the code is changed